Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -240,14 +280,44 @@ open class ChatMessageListVC: _ViewController,
         )
     }
 
-    /// Set the visibility of `scrollToBottomButton`.
+    /// Set the visibility of `scrollToLatestMessageButton`.
+    @available(*, deprecated, message: "use updateScrollToBottomButtonVisibility(animated:) instead.")
+    open func setScrollToLatestMessageButton(visible: Bool, animated: Bool = true) {
+        updateScrollToBottomButtonVisibility()
+    }
+
+    func updateScrollDependentButtonsVisibility(animated: Bool = true) {
+        updateScrollToBottomButtonVisibility(animated: animated)
+        updateJumpToUnreadMessagesVisibility(animated: animated)
+    }
+
+    /// Set the visibility of `scrollToLatestMessageButton`.
     open func updateScrollToBottomButtonVisibility(animated: Bool = true) {
-        let isVisible = isScrollToBottomButtonVisible
-        if isVisible { scrollToBottomButton.isVisible = true }
+        updateVisibility(
+            for: scrollToBottomButton,
+            isVisible: isScrollToBottomButtonVisible,
+            animated: animated
+        )
+    }
+
+    open func updateJumpToUnreadMessagesVisibility(animated: Bool = true) {
```

Comment:
I think it important that we add the "Button" naming, so that is clear we are talking about the button. We can also add a document comment like the one in `updateScrollToBottomButtonVisibility`
```suggestion
    open func updateJumpToUnreadButtonVisibility(animated: Bool = true) {
```